### ISSUE TITLE
Add 'pug' template engine (renamed from Jade)

### ISSUE THREAD

**Issue by piknik** [tj/consolidate.js#231]
I've had to switch some of my configuration due to [this post](https://github.com/pugjs/jade/issues/2184).

```
Module build failed: Error: Template engine 'pug' isn't available in Consolidate.js
```

Now, I cannot build 'pug' templates although I have pug installed in npm. Even though pug may not be used until Jade has completed its transition to pug, it needs to be implemented as Jade will soon not exist.

**Comment by sigo** [comment]
Huh, it's really annoying. Guys, could you fix this?

**Comment by stealthybox** [comment]
@sigo PR #232 fixes this.
I think we're just pending a design decision.

**Comment by mhumeSF** [comment]
:+1:

**Comment by 2ix** [comment]
@stealthybox any idea when it will be merged in?

**Comment by stealthybox** [comment]
@2ix 
Haven't received further follow up.
@BridgeAR / @tj 
Could we merge #232? Seems like the community has a need -- I certainly do.
Thanks much!

**Comment by tj** [comment]
hmm maybe we need a new maintain

**Comment by stealthybox** [comment]
@tj
I'm feeling some imposter syndrome, but I could read a PR a week.
not completely familiar with the codebase.

**Comment by doowb** [comment]
The PR's been merged. I'll try to get some more things merged in and publish a new version.

**Comment by doowb** [comment]
Published in `0.14.1`
